Homework_4/Repositories/CustomerRepository.cs

```diff
@@ -0,0 +1,37 @@
+using Homework_4.DbModels;
+using Homework_4.Provider;
+using Microsoft.EntityFrameworkCore;
+
+namespace Homework_4.Repositories;
+
+public class CustomerRepository : Repository
+{
+    public CustomerRepository(string connectionString, PurchaseDbContext dbProvider) : base(connectionString, dbProvider)
+    {
+    }
+
+    public async Task CreateCustomer(DbCustomer customer)
+    {
+        await DbProvider.AddAsync(customer);
```

Comment:
nah this is actually pointless
![image](https://github.com/Glunky/Lanit_HW_Duboyskiy_I_V/assets/101804724/d954bb30-f627-4e85-ae14-6c8da8c3288b)